apps/heft/UPGRADING.md

```diff
@@ -2,41 +2,89 @@
 
 ### Heft 0.32.0
 
-This release of Heft removed the Jest plugin from the `@rushstack/heft` package
-and moved it to it's own package (`@rushstack/heft-jest-plugin`). To re-include
-Jest support in a project, include a dependency on `@rushstack/heft-jest-plugin`
-and add the following option to the project's `config/heft.json` file:
-
-```JSON
-{
-  "heftPlugins": [
-    {
-      "plugin": "@rushstack/heft-jest-plugin"
-    }
-  ]
-}
-```
+Breaking change for Jest:  This release of Heft enables rig support for Jest config files.
+It also reduces Heft's installation footprint by removing the Jest plugin from `@rushstack/heft`
+and moving it to its own package `@rushstack/heft-jest-plugin`.  As a result, Jest is now
+disabled by default.
+
+To reenable Jest support for your project, follow these steps:
+
+1. Add the `@rushstack/heft-jest-plugin` dependency to your project's **package.json** file.
+
+2. If you are using `@rushstack/heft-node-rig` or `@rushstack/heft-web-rig`, the Jest
+   plugin is already enabled.  Skip to step 4.
+
+3. Load the plugin by adding this setting to your `config/heft.json` file:
+
+   ```js
```

Comment:
```json ?

Reply:
This is how ` ```json ` handles comments:

```json
{
  // comment
}
```

Thus I always use ` ```js ` for JSON.

Reply:
Ahh that's interesting didn't know that. Makes sense.